src/VisualQueryBuilder/components/LabelParamEditor.tsx

```diff
@@ -58,3 +58,5 @@ async function loadGroupByLabels(query: VisualQuery, datasource: DataSourceApi,
     value: x,
   }));
 }
+
+export const toOption = (value: QueryBuilderOperationParamValue | undefined): SelectableValue<QueryBuilderOperationParamValue | undefined> => ({ label: value?.toString(), value });
```

Comment:
No real need to export this, right?

Reply:
Fixed in https://github.com/grafana/grafana-experimental/pull/103/commits/b849a212e2e982ec3771d5e8ef02a57479dd2379